x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -8019,4 +8017,4 @@
     "xpack.watcher.watchActionsTitle": "满足后将执行 {watchActionsCount, plural, one{# 个操作} other {# 个操作}}",
     "xpack.watcher.watcherDescription": "通过创建、管理和监测警报来检测数据中的更改。"
   }
-}
+}
```

Comment:
looks like the new line may have been stripped

Reply:
Put the final new line back in.